requirements.txt

```diff
@@ -3,7 +3,7 @@ numpy~=1.26.4
 scipy~=1.13.1
 pandas~=2.2.2
 sympy~=1.12
-voila~=0.5.7
```

Comment:
0.5.7 is working on my end. Why downgrade?

Reply:
Ah ok i see because of MathJax issue. For me on 0.5.7 everything was fine. can you double-check?